### ISSUE TITLE
Blinking Cursors

### ISSUE THREAD

**Issue by MarjaE2** [plateaukao/einkbro#30]
By default, Android enables blinking cursors. I have neuro issues where blinking cursors, other flashing, and other animation can screw with my eyesight and my brain.

I have found 2 solutions--

Either stop blinking cursors, or use white text on black backgrounds for the affected fields on e-ink screens.

I don't think Android allows apps to stop blinking cursors, and my device uses Android 8.1 so it can't use a system setting in Android 10+. An app setting would be helpful.

**Comment by plateaukao** [comment]
Hi, do you mean input text in web content? Did you find any Open Source browser that support disabling curosr blinking? If so, I could take a look at how it is imolemented. If not, I may not be able to help you on this issue.

**Comment by MarjaE2** [comment]
Firefox has an about:config setting: ui.caretBlinkTime

Android apps can use various options to hide the cursor, but these don't seem to support showing it without blinking it.

https://code2care.org/pages/android-edittext-cursor-disable

https://stackoverflow.com/questions/5582702/disable-edittext-blinking-cursor

**Comment by plateaukao** [comment]
1. it looks like Firefox does not use Android's WebView component. It has its own imolementation, and the configuration of caretBlinkTime is implemented in C++ layer, which I don't know how it can be applied or implemented in Android built-in WebView.
2. For links you provided, they are for Android EditText Views. As I found, on Hisense Eink device, and Onyx Boox device, system manufacturer already made it not blink. I checked the cursor when trying to input url in the bottom toolbar, it is already static.
